.github/workflows/make.yml

```diff
@@ -30,4 +31,10 @@ jobs:
         sudo make PREFIX=/usr install
```

Comment:
You can avoid using `sudo` here by using `DESTDIR`, like so `make DESTDIR=destdir install`, which will install it under `$PWD/destdir`